### ISSUE TITLE
Fine-tuning: llama-2-13b-chat

### ISSUE THREAD

**Issue by llmwesee** [h2oai/h2ogpt#723]
For fine-tuning of the large language models (llama2), what should be the format(.text/.json/.csv) and structure (like should be an excel or docs file or prompt and response or instruction and output) of the training dataset? And also how to prepare or organise the tabular dataset for training purpose?

**Comment by pseudotensor** [comment]
Best to try https://github.com/h2oai/h2o-llmstudio for fine-tuning.

**Comment by llmwesee** [comment]
I tried llmstudio for fine-tuning & it's amazing.  I just want to know about data preparation steps for llama-2-13b-chat-hf
model like what is the ideal document format structure and what kind of prompt & instruction structure in it for Q/A use case. And how many minimum examples required for fine-tuning llama-2-13b-chat-hf model.

Previously I used 2000 pairs of instruction & output in excel format and convert it in .csv for fine-tuning using llmstudio UI interface but it doesn't work at all.
